.env

```diff
@@ -1,10 +1,12 @@
 REDIS_PORT=6380
 POSTGRES_PORT=5432
-GOLEM_SERVICE_HTTP_PORT=9881
-GOLEM_SERVICE_GRPC_PORT=9090
+GOLEM_GATEWAY_PORT=9881
```

Comment:
since docker image and service name is `golem-router`, i think we should rename variable to `GOLEM_ROUTER_HTTP_PORT`